src/ArborX_LinearBVH.hpp

```diff
@@ -88,6 +88,13 @@ class BasicBoundingVolumeHierarchy
                   std::forward<View>(view), std::forward<Args>(args)...);
   }
 
+  template <typename Predicate, typename Callback>
+  KOKKOS_FUNCTION void kernel_query(Predicate const &predicate,
```

Comment:
Why not just `query`?

Reply:
Any prior known practice to prepend `kernel_`?

Reply:
I'm open to other naming suggestions but that was the first one that came to mind.